### PR TITLE
Create issue templates and security policy

### PR DESCRIPTION
**Pull request by sfulham**

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Security Policy
+
+## Reporting a Vulnerability
+
+To report a vulnerability, create an issue with details on the vulnerability and how to exploit it, or create a pull request with a fix for the vulnerability.
```
Comment (RealOrangeOne) [inline SECURITY.md]: Opening a public issue for a security vulnerability is generally a very bad idea (https://en.wikipedia.org/wiki/Responsible_disclosure).

Disclosures like this need to be private at first. Best way to do that would probably be via https://selfhosted.show/contact.

File: .github/ISSUE_TEMPLATE/vulnerability-report.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+---
```
Comment (RealOrangeOne) [inline .github/ISSUE_TEMPLATE/vulnerability-report.md]: Can also remove this

Comment (sfulham) [reply .github/ISSUE_TEMPLATE/vulnerability-report.md]: That was automatically added by github

Comment (sfulham) [reply .github/ISSUE_TEMPLATE/vulnerability-report.md]: oh wait nvm

**Review by RealOrangeOne — APPROVED** [review]

**Pull request merged**